src/views/Layout.jsx

```diff
@@ -1,8 +1,7 @@
+/* eslint-disable jsx-a11y/anchor-is-valid */
```

Comment:
This was needed bc the git hooks did not let me commit.

Once the issue 3 is complete we can guide the collabies to delete this.

Reply:
Thanks for adding this!

Reply:
Revert changes look great! 🚀